File: Regression/EP_Web/Vanity/vanity_check.py
from selenium import webdriver
from selenium.webdriver.common.keys import Keys
from selenium.webdriver.support.select import Select
import time
import xlrd
import os

if os.environ.get('USE_PHANTOM'):
    print("using PhantomJS")

    driver = webdriver.PhantomJS()
    driver.implicitly_wait(5)
    driver.set_window_size(1120, 550)
else:
    print("using Firefox")
    driver = webdriver.Firefox()

driver.get("http://www.pt.energypluscompany.com/17save")
time.sleep(4)

if  "Not Found" in driver.page_source:
    print ("failed")
else:
    print ("passed")

driver.get("http://www.pt.energypluscompany.com/17ave")
time.sleep(4)

if  "Not Found" in driver.page_source:
    print ("passed")
else:
    print ("failed")
	
	
### Close Session
driver.close()

File: Regression/EP_Web/Vanity/vanity_check.rb
require 'rubygems'
require 'watir-webdriver'
require 'win32ole' 
require 'csv'

browser = Watir::Browser.new :firefox
browser.goto 'http://www.pt.energypluscompany.com/17save'

if
browser.text.include? 'Not Found'
print "Vanity Landing Page - Failed", "\n"
else
print "Vanity Landing Page - Passed", "\n"
end

browser.goto 'http://www.pt.energypluscompany.com/02402'
if
browser.text.include? 'Not Found'
print "Not Found - Passed", "\n"
else
print "Not Found - Failed", "\n"
end

browser.close